drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -410,11 +410,11 @@ def form_rule(id:)
       Hmis::Form::Instance.find_by(id: id)
     end
 
-    field :auto_exit_configs, Types::HmisSchema::AutoExitConfig.page_type, null: false
```

Comment:
This is OK for this field, but for future reference:

If you need to remove a field or query that's actively being used by the interface in the previous release (release-105), we want to deprecate the field rather than removing it. (By adding arg `deprecation_reason: 'something'`) https://graphql-ruby.org/fields/introduction.html#field-documentation

This is because when a deploy occurs while someone is using the application, their "old" frontend starts talking to the new backend version, while they still have that frontend js cached in their browser. So the user would be running release-105 on the frontend, and release-106 on the backend. (Even though the deploys happen at the same time, this situation still comes up because of the browser caching). So - in that scenario, if the frontend tries to request a field or query that no longer exists, they will see an error. For that reason we are starting to try to make API changes backwards-compatible as much as possible. Deprecated fields can be removed in the following release (we don't really have a process for that yet though)

The reason I say don't worry about it here, is because GR Devs are the only users that currently have access to this interface. Other users won't be making this query, so they won't hit the above scenario. Just wanted to explain that! [this ticket](https://www.pivotaltracker.com/story/show/184908546/comments/240087679) tracks some of the thinking about this over time.